### ISSUE TITLE
Linux: tags not displaying correctly

### ISSUE THREAD

**Issue by linuzzz** [laurent22/joplin#8000]
The very odd bug discussed in #7897 and #7506 is still present in 2.10.11 but not in the list of notebooks, only in the list of tags. See the attachment. I'm using Fedora 37 with Gnome. Thanks.
![jop](https://user-images.githubusercontent.com/10906359/228936971-84c3aa60-482e-4dcb-b999-881a394c0417.png)

**Comment by linuzzz** [comment]
I just discovered that logseq has the same bug, see the attachment, so i'm not an expert but the issue is not in the application but in the rendering library used by the application (as you alredy highlighted). Maybe the best option could be to find a working custom css? Thanks

![logseq](https://user-images.githubusercontent.com/10906359/229026159-02dcddd4-ee90-43e1-84b0-368dc0370905.png)

**Comment by Janaki4** [comment]
Hey , Can you please guide me how to run this project on my PC . I'm using windows . I'm stuck at build ing and running the project (Server and desktop client). any help would be helpful.

**Comment by cas--** [comment]
I have just finished creating a workaround for the same issue in [Logseq #7233](https://github.com/logseq/logseq/issues/7233). The conclusion is an Electron (Chromium) Linux Intel GPU rendering problem that really should be reported upstream.

Although setting the opacity does workaround the issue it is at the slight expense of a brighter UI for Linux users. After some further digging I found that the fractional opacity setting is creating a stacking context for the icon and text elements resulting in a rendering bug. This can be worked around by forcing a separate stacking context for the text elements.

I have submitted a PR but here is the change for using in `userchrome.css`:

```css
.list-item .title, .list-item .tag-label{
    position: relative;
}
```
